week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -16,15 +19,7 @@ const user: any = {
   },
 }
 
-function hasAddress(user: any): any {
-  return Boolean(user.address)
-}
-
-function hasGivenAge(requiredAge: any): any {
-  return (user: any): any => user.age >= requiredAge
-}
-
-const isAdult = hasGivenAge(18)
+const user: userClass = new userClass(userT)
 
-console.log(`User ${user.name} is ${isAdult(user) ? 'adult' : 'minor'}`)
-console.log(`and has${hasAddress(user) ? '' : ' no'} address`)
+console.log(`User ${user.user.name} is ${user.isAdult(18) ? 'adult' : 'minor'}`)
```

Comment:
Zmieniona została przez ciebie logika do `isAdult`.
Teraz moim zdaniem jakkolwiek zmienisz wiek `user`'a to i tak zawsze będzie `adult` bo warunek:
`user.isAdult(18)` to funkcja - wiec to zawsze `thruty`


Brakuje mi tutaj kodu do uruchamiania tego `main.ts` w projekcie :)